src/Microsoft.VisualStudio.Editors/SettingsDesigner/SettingsSingleFileGeneratorBase.vb

```diff
@@ -570,17 +567,12 @@ Namespace Microsoft.VisualStudio.Editors.SettingsDesigner
         ''' Creates a string representation of the full-type name give the project's root-namespace, the default namespace
         ''' into which we are generating, and the name of the class
         ''' </summary>
-        ''' <param name="projectRootNamespace">project's root namespace (may be String.Empty)</param>
         ''' <param name="defaultNamespace">namespace into which we are generating (may be String.Empty)</param>
         ''' <param name="typeName">the type of the settings-class we are generating</param>
-        Private Shared Function GetFullTypeName(projectRootNamespace As String, defaultNamespace As String, typeName As String) As String
+        Private Shared Function GetFullTypeName(defaultNamespace As String, typeName As String) As String
```

Comment:
Does this fixes the issue in VB ? does it work for C# ? the namespace might be coming from CPS.

if you hover the mouse over the class definition, does it show the correct namespace ?

Reply:
it fixes the issue in VB, still works for c#.

VB:
![image](https://user-images.githubusercontent.com/20359921/180494805-d004c211-76db-4091-ab77-62068c346caf.png)

C#
![image](https://user-images.githubusercontent.com/20359921/180495033-6aab03b0-e9e6-4357-9045-da4a2eeab05b.png)